### PR TITLE
[AS-83] Screen for room creation

### PR DESCRIPTION
**Pull request by mslup**
`/creator` endpoint

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+    "editor.formatOnPaste": true,
+    "editor.formatOnSave": true
+}
```
Comment (andrmdmd) [inline .vscode/settings.json]: Please do not upload your local configs (you can add it to .gitignore)

File: src/components/ValueSlider/ValueSlider.css
Comment (andrmdmd) [inline src/components/ValueSlider/ValueSlider.css:1]: shouldn't folder name be 'valueSlider'?

File: src/components/button/Button.tsx [hunk 2]
```diff
@@ -10,21 +10,13 @@ type ButtonProps = {
 
 const Button = ({
   children,
-  onClick = () => {},
+  onClick = () => { },
```
Comment (andrmdmd) [inline src/components/button/Button.tsx]: linting

File: src/components/properties/Properties.tsx [hunk 3]
```diff
@@ -1,64 +1,72 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 // todo for the future^
+import './Properties.css';
```
Comment (andrmdmd) [inline src/components/properties/Properties.tsx]: please import css files last to match convention of other files

**Review by andrmdmd — COMMENTED** [review]
image

Is this the expected behavior for now?

File: .vscode/settings.json (hunk 1 above)
Comment (mslup) [reply .vscode/settings.json]: omg this is humbling

**Comment by mslup** [comment]
> Is this the expected behavior for now?

yes

File: src/components/button/Button.tsx (hunk 2 above)
Comment (mslup) [reply src/components/button/Button.tsx]: ESLint doesn't report an issue

File: src/components/ValueSlider/ValueSlider.css
Comment (mslup) [reply src/components/ValueSlider/ValueSlider.css:1]: fair

File: src/components/button/Button.tsx (hunk 2 above)
Comment (andrmdmd) [reply src/components/button/Button.tsx]: I report it 🤭

Comment (mslup) [reply src/components/button/Button.tsx]:
```suggestion
  onClick = () => {},

```

**Review by andrmdmd — APPROVED** [review]

**Comment by mslup** [comment]
Accidentally also merged model rotations.

**Pull request merged**